### ISSUE TITLE
Difficulty with the install

### ISSUE THREAD

**Issue by Ashkeelun** [jakeharding/repo-health#56]
So I've been trying for a while now and have had a lot of problems getting it installed on my computer.

First it docker isn't compatible with Windows Home edition, so I got Windows Pro installed and then I ran into this error:

![image](https://cloud.githubusercontent.com/assets/14626151/25047273/5b47f9e0-20fb-11e7-8502-1bfe778909c0.png)

I read through your documentation but didn't find any manual installation instructions. If I could get a list of all the python dependencies I think I could manage it with a bit of work but at this point I'm kind of dead in the water as my current computer doesn't really support virtualization.

**Comment by bparish628** [comment]
Take a look at [Other Installation](https://github.com/jakeharding/repo-health/blob/dev/docs/Other%20Installation.md).
It's not in master yet (still in dev), but that is how to install everything locally.

**Comment by Ashkeelun** [comment]
I have successfully made it this far in the full install doc

![image](https://cloud.githubusercontent.com/assets/14626151/25049658/86c57eaa-2108-11e7-95eb-33f128f9b86a.png)

When I run these initial migrations should I get a CommandError?
![image](https://cloud.githubusercontent.com/assets/14626151/25049680/a82f327a-2108-11e7-9fc4-25b6426d4f87.png)

**Comment by bparish628** [comment]
I don't actually think you have to do that 😬 . Try just running a `python manage.py migrate`.

**Comment by Ashkeelun** [comment]
Okay it looks like that worked.
does the test use a different database from msr14?

![image](https://cloud.githubusercontent.com/assets/14626151/25050012/5bab8fc8-210a-11e7-8264-4566cb5a8db2.png)

**Comment by bparish628** [comment]
@jakeharding would know better, but from my understanding it creates a test database with the same settings from local_settings.py (with _test appended). Is your local_settings.py setup?

**Comment by jakeharding** [comment]
You will need to setup a test database for the tests as covered in the readme.

**Comment by Ashkeelun** [comment]
Yeah I don't think the user has access to do that but I could create the database

**Comment by germonprez** [comment]
You may be able to use

opendata.missouri.edu
User msr
Pass ghtorrent
DB msr

**Comment by jakeharding** [comment]
I am not sure what you mean by the user doesn't have access but you create another database in the fashion as you did for production but prepend the name with 'test_'.

**Comment by Ashkeelun** [comment]
I tried creating a test_msr14 schema that was still referencing the msr14 shema so I changed the local_settings.py to the test_msr14 database. It complained that there were no tables so I ran all the migrations then it still errored out so I tries changing settings to use the database that @germonprez provided above and I got the same error I was getting before which was this.

![image](https://cloud.githubusercontent.com/assets/14626151/25050605/10226140-210e-11e7-82af-ac3846e8604a.png)

I've spent over 15 hours trying different things to get this to run on my hardware.

Also I began reading down further, is the npm start a node.js server?

**Comment by jakeharding** [comment]
Not exactly. You will need to create a whole new database for testing and populate it with same data as your production database. You will not need to make any changes to your local settings because Django will look for a database named with test_ followed by you database name in the local settings.

**Comment by Ashkeelun** [comment]
I've tried, this is what I get.

![image](https://cloud.githubusercontent.com/assets/14626151/25050956/de985fd8-210f-11e7-9216-f676f44e122c.png)

**Comment by jakeharding** [comment]
Normally the host is set to local host. Does the dev server start when you run manage.py runserver?

**Comment by Ashkeelun** [comment]
yes
![image](https://cloud.githubusercontent.com/assets/14626151/25051084/7ff70e7e-2110-11e7-9530-08dc768e9908.png)


here I recreated test_msr14

![image](https://cloud.githubusercontent.com/assets/14626151/25051056/5a711bd6-2110-11e7-89dc-33145bd707d9.png)

**Comment by jakeharding** [comment]
It looks like the connection is made but no tables have been created. Are you able to view data in the test database in your mysql workbench?

**Comment by Ashkeelun** [comment]
Yes
![image](https://cloud.githubusercontent.com/assets/14626151/25051365/d2f908c4-2111-11e7-8e2e-b7783ceab8f5.png)

**Comment by jakeharding** [comment]
Those are all of the Django tables but none of the msr14 tables. You will need to run msr14 SQL script to create the tables and insert the data.

**Comment by jakeharding** [comment]
@Ashkeelun Any luck?

**Comment by Ashkeelun** [comment]
heh.. no I'm not really sure where that script is located. I couldn't find it in you're repo, and the computer I had got everything setup for this is at my work, and I went home for the day.

**Comment by jakeharding** [comment]
You will have to download it from ghtorrent and unpack it. The link is in the readme.

**Comment by Ashkeelun** [comment]
Okay, well I've decided to attempt using AWS's Docker Container service, because no mater what I'm running into some form of incompatibility.
